source/api/presentation/3.0/index.md

```diff
@@ -483,16 +483,17 @@ The value _MUST_ be an array of strings, taken from the table below or a URI.
 
 > | Value | Description |
 | ----- | ----------- |
-| `individuals` | Valid on Collection, Manifest, Sequence and Range. For Collections with this `behavior`, each of the included Manifests are distinct objects. For Manifest, Sequence and Range, the included Canvases are distinct views, and _SHOULD NOT_ be presented in a page-turning interface. This is the default `behavior` if none are specified. |
-| `paged` | Valid on Manifest, Sequence and Range, which include Canvases that have at least `height` and `width` dimensions. Canvases included in resources with this `behavior` represent pages in a bound volume, and _SHOULD_ be presented in a page-turning interface if one is available.  The first canvas is a single view (the first recto) and thus the second canvas likely represents the back of the object in the first canvas. If this is not the case, see the "non-paged" `behavior`. |
+| `auto-advance` | Valid on Collection, Manifest, Sequence and Canvas, that include or are Canvases with at least the `duration` dimension. When the client reaches the end of a Canvas with a duration dimension that has (or is within a resource that has) this `behavior`, it _SHOULD_ immediately proceed to the next Canvas and render it. If there is no subsequent Canvas in the current context, then this `behavior` should be ignored. When applied to a Collection, the client should treat the first Canvas of the next Manifest as following the last Canvas of the previous Manifest, respecting any `startCanvas` specified. |
```

Comment:
> Valid on Collection, Manifest, Sequence and Canvas, that include or are Canvases

Typo? Shouldn't "Collection, Manifest, Sequence and Canvas" each be pluralized? We use this construction elsewhere as well and it reads awkwardly (e.g. `repeat`, `paged`)

Reply:
:+1: I thought this at the time, but it wasn't part of the issue so I didn't change it.  Changing :)